assets/queries/terraform/azure/postgre_sql_server_infrastructure_encryption/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "6425c98b-ca4e-41fe-896a-c78772c131f8",
+  "queryName": "PostgreSQL Server Infrastructure Encryption",
```

Comment:
Maybe we can rename it for something more intuitive like `"queryName": "PostgreSQL Server Infrastructure Encryption Disabled"`